odoo/tools/template_inheritance.py

```diff
@@ -159,7 +159,13 @@ def extract(spec):
                     # fix, this solution was chosen: the location is marked
                     # with a "ProcessingInstruction" which will not impact
                     # the "Element" structure of the resulting tree.
-                    if inherit_branding:
+                    # Exception: if we happen to replace a node that already
+                    # has xpath branding (root level nodes), do not mark the
+                    # location of the removal as it will mess up the branding
+                    # of siblings elements coming from other views, after the
+                    # branding is distributed (and those processing instructions
+                    # removed).
+                    if inherit_branding and not node.get('data-oe-xpath'):
```

Comment:
Shit, you're probably right. I removed data-oe-field-xpath in later versions, forgot it still existed in 13.0.

Reply:
@rdeodoo Adapted with https://github.com/odoo/odoo/compare/a274b12ae566ddb923c52d29ae8aee0209c115b8..fef887f6c356dc2e40a232e610745a2022b36ba7 -> I changed the last test also so that it failed without the data-oe-field-xpath check. As explained by the commit message, the 3 added tests are redundant so I don't think it was worth a 4th test to make a difference between t-field or not (especially since it will be the same in the version where data-oe-field-xpath was removed).

I let you r+ if you are ok :+1: